### PR TITLE
Adding no-switch-case-fall-through rule to tslint.

### PR DESCRIPTION
**Pull request by dlebu**
Better safe than sorry.

### CODE REVIEW

**Comment by digeff** [comment]
:+1:

File: src/common/log/logHelper.ts [hunk 1]
```diff
@@ -75,15 +75,14 @@ export class LogHelper {
         }
 
         switch (error.errorLevel) {
-            case InternalErrorLevel.Error: {
+            case InternalErrorLevel.Error:
                 // Encode the error code to a four-char code - ex, 0198
                 let errorCodeString = (LogHelper.ERROR_CODE_WIDTH + error.errorCode).slice(-LogHelper.ERROR_CODE_WIDTH.length);
```
Comment (MSLaguana) [inline src/common/log/logHelper.ts]: Won't removing the `{}` change the scoping of the let statement? Is that deliberate?

Comment (dlebu) [inline src/common/log/logHelper.ts]: It is deliberate - tslint would complain - it checks for break or return before case statements.

Comment (meenakb) [inline src/common/log/logHelper.ts]: Interesting. :+1:

**Comment by MSLaguana** [comment]
Looks good to me

**Pull request merged**